### PR TITLE
Running commands silently

### PR DESCRIPTION
**Pull request by shigupt202**
**Task name**: HelmDeployV0

**Description**: Running `helm get manifest` and `kubectl get pods -o json` silently to avoid revealing secrets and lengthy logs

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) (Updated)

**Attached related issue:** (Y/N) https://github.com/microsoft/azure-pipelines-tasks/issues/13175

**Checklist**:
✔️  Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
✔️  Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/HelmDeployV0/task.loc.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 0,
     "Minor": 172,
-    "Patch": 0
+    "Patch": 1
```
Comment (ds-ms) [inline Tasks/HelmDeployV0/task.loc.json:17]: This patch number might conflict

Comment (shigupt202) [reply Tasks/HelmDeployV0/task.loc.json:17]: Why exactly? The current task version in master is 172.0

Comment (shigupt202) [reply Tasks/HelmDeployV0/task.loc.json:17]: Got it. Because of the other unmerged PR.

**Comment by vaclavbroz** [comment]
Now the helm task will run kubectl cluster-info which will result in a forbidden warning because cluster-info may not be accessible for the given service connection, e.g. Error from server (Forbidden): services is forbidden: User "system:serviceaccount:xxx-sa:xxx cannot list resource "services" in API group "" in the namespace "kube-system"

**Pull request merged**